apps/emqx_durable_storage/src/emqx_ds_replication_layer.erl

```diff
@@ -500,3 +471,244 @@ do_get_delete_streams_v4(DB, Shard, TopicFilter, StartTime) ->
 
 list_nodes() ->
     mria:running_nodes().
+
+%% TODO
+%% Too large for normal operation, need better backpressure mechanism.
+-define(RA_TIMEOUT, 60 * 1000).
+
+-define(SAFERPC(SERVER, EXPR), make_safe_rpc(SERVER, fun() -> EXPR end)).
```

Comment:
This is not the safest RPC, since it creates a closure object. Upgrading code will break it. It's safer to use something like `{fun my_fun/1, ArgsList}` or an mfa.